### PR TITLE
Validate lookup table names

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?189828

@nickpell / @TylerSheffels

### CODE REVIEW

**Comment by TylerSheffels** [comment]
:+1: After tests

File: corehq/apps/fixtures/templates/fixtures/partials/edit_table_modal.html [hunk 1]
```diff
@@ -18,10 +18,10 @@
                 <thead>
                     <tr>
                         <th class="span10">
-                            field_name
+                            Name
```
Comment (czue) [inline corehq/apps/fixtures/templates/fixtures/partials/edit_table_modal.html]: missing trans?

File: corehq/apps/fixtures/templates/fixtures/manage_tables.html [hunk 2]
```diff
@@ -91,7 +91,7 @@ <h4 class="pull-right">
                             <td>
                                 <button class="btn" data-bind="click: startEditing, visible: saveState() === 'saved'">
                                     <i class="icon-edit"></i>
-                                    Edit Type
+                                    Edit
```
Comment (czue) [inline corehq/apps/fixtures/templates/fixtures/manage_tables.html]: trans

File: corehq/apps/fixtures/templates/fixtures/partials/edit_table_modal.html [hunk 3]
```diff
@@ -18,10 +18,10 @@
...
                         </th>
                         <th class="span2">
-                            Delete Field?
+                            Delete?
```
Comment (czue) [inline corehq/apps/fixtures/templates/fixtures/partials/edit_table_modal.html]: (and here)

**Comment by orangejenny** [comment]
added translations and fixed some weird markup ("weird" => inline styling, using `.modal-body` and `.modal-footer` outside of modals)

**Comment by orangejenny** [comment]
to show that the markup changes are minor & reasonable, this is the old:
screen shot 2015-12-29 at 6 10 33 pm

and this is the new:
screen shot 2015-12-29 at 6 10 44 pm

**Comment by orangejenny** [comment]
tests pass, should be good to go

**Comment by czue** [comment]
looking good!

**Pull request merged**